whatsapp_bot.py

```diff
@@ -88,25 +92,30 @@ def generate_reply(incoming_message: str, sender_contact: str, receiver_contact:
         Response text
     """
     text_message = incoming_message.lower()
+
     # 'explain' response
     if text_message[0:7] == "explain":
         response = ActivityGenerator.generate(
             model=LLM,
             temperature=TEMPERATURE,
-            messages=[ELI3_MESSAGES],
+            messages=[ELI3_MESSAGES.copy()],
```

Comment:
@kstathou FYI: Had to make this a copy otherwise the app, after the first response, would respond with the same message for any other subsequent `text_message`